client/lobby/RandomMapTab.cpp

```diff
@@ -125,6 +121,35 @@ RandomMapTab::RandomMapTab():
 	const JsonNode config(ResourceID("config/widgets/randomMapTab.json"));
 	build(config);
 	
+	//set combo box callbacks
+	if(auto w = widget<ComboBox>("templateList"))
```

Comment:
What will happen if player with vcmi-extras from 1.3 uses RMG in 1.4?

Reply:
old mod has compatibility restriction. If it not updated at all, there will be some missing elements